cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -340,12 +340,21 @@ endif()
 if(DEFINED ENV{ARROW_JEMALLOC_URL})
   set(JEMALLOC_SOURCE_URL "$ENV{ARROW_JEMALLOC_URL}")
 else()
+  # XXX jemalloc has two possible download URLs.  This one includes the configure
+  # script.  The one in thirdparty/versions.txt doesn't.
```

Comment:
We've pushed https://github.com/apache/arrow/commit/7d63dfe0709201d26d3cc84186e69ec946236604 recently.
Can we remove this?

Reply:
Will do.